programs/clearing_house/src/lib.rs

```diff
@@ -682,16 +643,14 @@ pub mod clearing_house {
             now,
         )?;
 
-        validate!(
```

Comment:
dont we need a check that the market is active? should double check that we're doing the active check everywhere

Reply:
added